doc/mount_persistent_volume_howto.md

```diff
@@ -0,0 +1,125 @@
+# Mount Persistent Volume
+
+StarRocks Kubernetes Operator supports mounting persistent volumes to StarRocks FE and BE pods. If not specified, the
+operator will use emptyDir mode to store FE meta and BE data. **When container restarts, the data will be lost.**
+
+This document describes how to mount persistent volumes to StarRocks FE and BE pods. And There are two ways to mount
+persistent volumes to StarRocks FE and BE pods:
+
+1. Mounting persistent volumes to StarRocks FE and BE pods by the StarRocks CRD YAML file.
+2. Mounting persistent volumes to StarRocks FE and BE pods by Helm chart.
+
+> Note: Starrocks Operator will create a new PVC for each storageVolume. You should not create PVC manually.
+
+## 1. Mounting Persistent Volumes by StarRocks CRD YAML File
+
+If you want to use external storage to store FE meta and BE data for persistence, you can specify `storageVolumes` in
+the corresponding component spec.
+
+The following is an example of mounting persistent volumes to StarRocks FE and BE.
+
+```bash
+apiVersion: starrocks.com/v1
+kind: StarRocksCluster
+metadata:
+  name: kube-starrocks
+  namespace: starrocks
+  labels:
+    cluster: kube-starrocks
+spec:
+  starRocksFeSpec:
+    image: "starrocks/fe-ubuntu:3.1-latest"
+    replicas: 1
+    storageVolumes:
+    - name: fe-storage-meta
+      storageClassName: data-storage
+      storageSize: 10Gi
+      mountPath: /opt/starrocks/fe/meta
+    - name: fe-storage-log
+      storageClassName: data-storage
+      storageSize: 1Gi
+      mountPath: /opt/starrocks/fe/log
+  starRocksBeSpec:
+    image: "starrocks/be-ubuntu:3.1-latest"
+    replicas: 1
+    storageVolumes:
+    - name: be-storage-data
+      storageClassName: data-storage
+      storageSize: 10Gi
+      mountPath: /opt/starrocks/be/storage
+    - name: be-storage-log
+      storageClassName: data-storage
+      storageSize: 1Gi
+      mountPath: /opt/starrocks/be/log
+```
+
+Note that the specific `storageClassName` should be available in kubernetes cluster before enabling this storageVolume
+feature. If `StorageVolume` info is not specified in CRD spec, the operator will use emptydir mode to store FE meta and
+BE data.
```

Comment:
And a FAQ or Warning notice, if the free space of fe-meta storage volume is less than 5GB, the FE service won't start.